File: Programar em C#/19-break-e-continue.cs
// Programar em Python #19 - break e continue

using System;
using System.Collections.Generic;

namespace Base {
    class BreakEContinue {
        public void Run() {
            int contador = 0;
            List<string> animais = new List<string>() {
                "Cão",
                "Gato",
                "Galinha",
                "Coelho",
                "Leão"
            };

            foreach (string animal in animais) {
                Console.WriteLine($"Animal: {animal}");

                if (animal == "Galinha")
                    break;
            }

            while (contador < 10) {
                contador++;

                if (contador == 5)
                    continue;

                Console.WriteLine($"Contador: {contador}");
            }
        }
    }
}

File: Programar em Java/19-break-e-continue.java
// Programar em Java #19 - break e continue

package com.caffeinealgorithm.programaremjava;

import java.util.List;
import java.util.ArrayList;

public class BreakEContinue {
    public void Run() {
        int contador = 0;
        List<String> animais = new ArrayList<>();

        animais.add("Cão");
        animais.add("Gato");
        animais.add("Galinha");
        animais.add("Coelho");
        animais.add("Leão");

        for (String animal : animais) {
            System.out.println(new StringBuilder().append("Animal: ").append(animal));

            if (animal == "Galinha")
                break;
        }

        while (contador < 10) {
            contador++;

            if (contador == 5)
                continue;

            System.out.println(new StringBuilder().append("Contador: ").append(contador));
        }
    }
}